styles/shinigami-eyes/catppuccin.user.css

```diff
@@ -0,0 +1,143 @@
+/* ==UserStyle==
+@name Shinigami Eyes Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/shinigami-eyes
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/shinigami-eyes
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/shinigami-eyes/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Ashinigami-eyes
+@description Soothing pastel theme for Shinigami Eyes
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document
+  domain(facebook.com),
+  domain(youtube.com),
+  domain(reddit.com),
+  domain(twitter.com),
+  domain(x.com),
+  domain(medium.com),
+  domain(disqus.com),
+  domain(tumblr.com),
+  domain(wikipedia.org),
+  domain(rationalwiki.org),
+  domain(cohost.org),
+  domain(bsky.app),
+  domain(anarchism.space),
+  domain(aus.social),
+  domain(c.im),
+  domain(chaos.social),
+  domain(eightpoint.app),
+  domain(eldritch.cafe),
+  domain(fosstodon.org),
+  domain(hachyderm.io),
+  domain(infosec.exchange),
+  domain(kolektiva.social),
+  domain(mas.to),
+  domain(masto.ai),
+  domain(chaosfem.tw),
+  domain(mastodon.art),
+  domain(mastodon.cloud),
+  domain(mastodon.green),
+  domain(mastodon.ie),
+  domain(mastodon.nz),
+  domain(mastodon.online),
+  domain(mastodon.scot),
+  domain(mastodon.social),
+  domain(mastodon.world),
+  domain(mastodon.xyz),
+  domain(mastodonapp.uk),
+  domain(meow.social),
+  domain(mstdn.ca),
+  domain(mstdn.jp),
+  domain(mstdn.social),
+  domain(octodon.social),
+  domain(ohai.social),
+  domain(pixelfed.social),
+  domain(queer.party),
+  domain(sfba.social),
+  domain(social.transsafety.network),
+  domain(tech.lgbt),
+  domain(techhub.social),
+  domain(toot.cat),
+  domain(toot.community),
+  domain(toot.wales),
+  domain(vulpine.club),
+  domain(wandering.shop),
+  domain(lgbtqia.space),
+  domain(threads.net),
+  domain(duckduckgo.com),
+  domain(bing.com),
+  domain(google.ar),
+  domain(google.at),
+  domain(google.be),
+  domain(google.ca),
+  domain(google.ch),
+  domain(google.co.uk),
+  domain(google.com),
+  domain(google.de),
+  domain(google.dk),
+  domain(google.es),
+  domain(google.fi),
+  domain(google.fr),
+  domain(google.is),
+  domain(google.it),
+  domain(google.no),
+  domain(google.pt),
+  domain(google.se)
```

Comment:
can we put the domains in quotes
e.g. `domain("google.it")`

Reply:
> can we put the domains in quotes
e.g. domain("google.it")

seems good, i'll do that